static/app/views/replays/detail/useActiveTabFromLocation.tsx

```diff
@@ -1,6 +1,6 @@
 import {isReplayTab, ReplayTabs} from '../types';
 
-const DEFAULT_TAB = ReplayTabs.PERFORMANCE;
+const DEFAULT_TAB = ReplayTabs.CONSOLE;
```

Comment:
Changed to Console tab because I initially deleted the Performance tab, but this will end up changing again probably. It was a bit awkward to have the 2nd tab selected, so whatever is our initial tab should be the leftmost tab.